src/app/applications/applications-menu/applications-menu.component.html

```diff
@@ -38,7 +38,7 @@
     <cbp-applications-search></cbp-applications-search>
 
 
-    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">
+    <div class="mat-menu-item cbp-applications-refresh" role="button" (click)="reloadApplicationsData($event)">
```

Comment:
Not sure this solves the issue.  Is the div focusable?  also, does it have the right keyboard actions?  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role

Reply:
It does not have keyboard. It should.

Reply:
Assigned  #21 to @jscharett